dkist/dataset/loader.py

```diff
@@ -45,45 +45,46 @@ def load_dataset(target):
     Examples
     --------
 
+    >>> import dkist
+
     >>> dkist.load_dataset("/path/to/VISP_L1_ABCDE.asdf")  # doctest: +SKIP
 
     >>> dkist.load_dataset("/path/to/ABCDE/")  # doctest: +SKIP
 
     >>> dkist.load_dataset(Path("/path/to/ABCDE"))  # doctest: +SKIP
 
-    >>> from sunpy.net import Fido, attrs as a
-    >>> import dkist.net
-    >>> search_results = Fido.search(a.dkist.Dataset("AGLKO"))   # doctest: +REMOTE_DATA
-    >>> files = Fido.fetch(search_results)   # doctest: +REMOTE_DATA
-    >>> dkist.load_dataset(files)   # doctest: +REMOTE_DATA
-    <dkist.dataset.dataset.Dataset object at ...>
-    This Dataset has 4 pixel and 5 world dimensions
+    >>> from dkist.data.sample import VISP_BKPLX  # doctest: +REMOTE_DATA
+    >>> print(dkist.load_dataset(VISP_BKPLX))  # doctest: +REMOTE_DATA
+    This VISP Dataset BKPLX has 4 pixel and 5 world dimensions and consists of 1700 frames
+    Files are stored in /home/drew/.local/share/dkist/VISP_BKPLX
```

Comment:
This wont work.

```suggestion
    Files are stored in /.../VISP_BKPLX
```

This should work (you can generally use Ellipsis to truncate output in the doctest and have it still pass)

Reply:
This is now upset because it's expecting Windows-directed slashes but I don't know if that's a thing we want to change (or indeed how to if we do).

Reply:
ah. you could just do: `...VISP_BKPLX` and then not have any slashes at all?